scripts/install/sabnzbd.sh

```diff
@@ -9,52 +9,79 @@
 #   changes/dates in source files. Any modifications to our software
 #   including (via compiler) GPL-licensed code must also be made available
 #   under the GPL along with build & install instructions.
-
-user=$(cut -d: -f1 < /root/.master.info)
-password=$(cut -d: -f2 < /root/.master.info)
-distribution=$(lsb_release -is)
-codename=$(lsb_release -cs)
-#latest=$(curl -s https://sabnzbd.org/downloads | grep -m1 Linux | grep download-link-src | grep -oP "href=\"\K[^\"]+")
-latest=$(curl -sL https://api.github.com/repos/sabnzbd/sabnzbd/releases/latest | grep -Po '(?<="browser_download_url":).*?[^\\].tar.gz"' | sed 's/"//g')
-latestversion=$(echo $latest | awk -F "/" '{print $NF}' | cut -d- -f2)
-
-. /etc/swizzin/sources/functions/pyenv
+#
+#shellcheck source=sources/functions/utils
 . /etc/swizzin/sources/functions/utils
+#shellcheck source=sources/functions/pyenv
+. /etc/swizzin/sources/functions/pyenv
+#
+user="$(_get_master_username)"
+password="$(_get_user_password "$user")"
+#
+latestversion=$(git ls-remote -t --sort=-v:refname --refs https://github.com/sabnzbd/sabnzbd.git | awk '{sub("refs/tags/", "");sub("(.*)(RC|Alpha|Beta|Final)(.*)", ""); print $2 }' | awk '!/^$/' | head -n1)
+latest="https://github.com/sabnzbd/sabnzbd/releases/download/${latestversion}/SABnzbd-${latestversion}-src.tar.gz"
 
-LIST='par2 p7zip-full python3-dev python3-setuptools python3-pip python3-venv libffi-dev libssl-dev libglib2.0-dev libdbus-1-dev'
-
-apt_install $LIST
+apt_install par2 p7zip-full python3-dev python3-setuptools python3-pip python3-venv libffi-dev libssl-dev libglib2.0-dev libdbus-1-dev
 
-python3_venv ${user} sabnzbd
+python3_venv "${user}" sabnzbd
 
-install_rar
+[[ "$(_os_distro)" == 'ubuntu' ]] && install_rar
+[[ "$(_os_distro)" == 'debian' ]] && _rar
 
 echo_progress_start "Downloading and extracting sabnzbd"
-cd /opt
+cd /opt || exit 1
 mkdir -p /opt/sabnzbd
-wget -O sabnzbd.tar.gz $latest >> $log 2>&1
-tar xzf sabnzbd.tar.gz --strip-components=1 -C /opt/sabnzbd >> ${log} 2>&1
+_cmd_log wget -O sabnzbd.tar.gz "${latest}"
+_cmd_log tar xzf sabnzbd.tar.gz --strip-components=1 -C /opt/sabnzbd
```

Comment:
Re: `_cmd_log`, I'd take that out of this PR so we can just merge at as this will need a discussion on how to do this right and the rest of this should go in first